iothub/device/src/TransportSettings/IotHubClientTransportSettings.cs

```diff
@@ -28,29 +27,27 @@ public abstract class IotHubClientTransportSettings
         /// The web proxy that will be used to connect to IoT hub using a web socket connection for AMQP, MQTT, or when using the
         /// HTTP protocol.
         /// </summary>
-        /// <value>
-        /// An instance of a class that implements <see cref="IWebProxy"/>.
-        /// </value>
         /// <remarks>
-        /// This setting will only be used when the client connects over web sockets or HTTPS.
+        /// If you wish to bypass OS-specified proxy settings, set this to <see cref="GlobalProxySelection.GetEmptyWebProxy()"/>.
         /// </remarks>
+        /// <seealso href="https://docs.microsoft.com/dotnet/api/system.net.http.httpclienthandler.proxy?view=net-6.0"/>
         /// <example>
-        /// To set a proxy you must instantiate an instance of the <see cref="WebProxy"/> class--or any class that derives from
-        /// <see cref="IWebProxy"/>. The snippet below shows a method that returns a device using a proxy that connects to localhost
-        /// on port 8888.
+        /// To set a proxy you must instantiate an instance of the <see cref="WebProxy"/> class--or any class that derives from <see cref="IWebProxy"/>.
+        /// The snippet below shows a method that returns a device using a proxy that connects to localhost on port 8888.
         /// <c>
-        /// static DeviceClient GetClientWithProxy()
+        /// static ServiceClient GetServiceClient()
```

Comment:
Copypaste error, this should be demonstrating device client code

Reply:
Fixed